src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -295,11 +295,12 @@ def init_filesystem_fields(partition)
 
       # @param fs [Filesystem::BlkFilesystem]
       def init_mount_options(fs)
-        if fs.mountpoint && !fs.mountpoint.empty?
-          @mount = fs.mountpoint
-          @mountby = fs.mount_by.to_sym
+        if !fs.mount_point.nil?
+          @mount = fs.mount_point.path
```

Comment:
Since we are dealing with so many mount-related things here, is it really a good idea to name a member variable @mount and not something more precise like @mount_path?

Reply:
Checking it with Imo, that names correspond to the names in the autoyast xml. For now, I would not change them because we would have to adapt a lot of code.

Reply:
OK then.